### PR TITLE
Added a docs folder and a sample index page.

### PR DESCRIPTION
**Pull request by meln1k**
In Github there is a feature for automatic docs generation based on `/docs` folder in master.:
image

Let's try it out.

JIRA issues: DCOS-47511

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Unified Scheduler Interface
+
+This repository is currently a work-in-progress.
+
+[Design principles behind USI](https://github.com/mesosphere/marathon-design/blob/master/unified-scheduler/index.md)
```
Comment (zen-dog) [inline docs/index.md]: `marathon-design` is a private repo

**Review by zen-dog — APPROVED** [review]
Let's try it, however `marathon-design` is a private repo

**Review by alenkacz — APPROVED** [review]
👍 🤷‍♀️ but having scaladocs is probably more important for a library than a pure docs page...

**Comment by jeschkies** [comment]
I don't think this is the way to go. How are we going to land doc changes?

**Pull request merged**